css-basics/css-basics_pink-box/css/styles.css

```diff
@@ -9,9 +9,14 @@ body {
 }
 
 /* put a selector here to select div elements (see index.html) */
-SELECTOR HERE {
+div {
   font-family: cursive;
   padding: 8px;
+  color: white;
+  background-color: hotpink;
+  height: 150px;
+  width: 150px;
+  border-radius: 20px;
```

Comment:
i think it should be 8px radius but as it says feel free to experiment that's probably also fine